statik/styles/_layout.styl

```diff
@@ -9,14 +9,15 @@ body
   color electromagnetic
   font-family 'Open Sans', sans-serif
   font-size 16px
-  line-height 1.4
+  line-height 1.5
 
 h1,
 h2,
 p
   font-weight 400
   margin 0
   padding 0
+  max-width 80ch
```

Comment:
@eubond Please add the CSS properties in alphabetical order.

Thanks for your contribution! 😉